vehicle/src/main/java/org/egov/vehicle/trip/validator/VehicleTripValidator.java

```diff
@@ -155,6 +155,12 @@ public void validateUpdateRecord(VehicleTripRequest request) {
 				if(request.getVehicleTrip().getTripEndTime() <= 0) {
 				throw new CustomException(VehicleTripConstants.INVALID_TRIP_ENDTIME, "Invalid Trip end time");
 			}
+			if (request.getVehicleTrip().getFstpEntryTime() <= 0 || request.getVehicleTrip().getFstpExitTime() <= 0

```

Comment:
remove the validation for TripEndTime or copy the value of FstpExitTime, as ui will be populating the dumpTime value to fstopExitTime instead of tripEndTime.